docs/community/contributing.rst

```diff
@@ -58,6 +58,7 @@ install Batavia's JavaScript dependencies, and compile the Batavia library:
 .. code-block:: bash
 
     $ npm install
+    $ npm install -g npm
```

Comment:
Good idea - but this line needs to be run before `npm install`; if you do it this way, you'll end up with the install being done with the stale version of npm.

If you reverse the order of line 60 and 61, it should be good to go.